web/src/features/hfiCalculator/components/DayHeaders.tsx

```diff
@@ -0,0 +1,44 @@
+import { TableCell } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { NUM_WEEK_DAYS } from 'features/hfiCalculator/constants'
+import { range } from 'lodash'
+import React from 'react'
+import { getPrepWeeklyDateRange } from 'utils/date'
+
+export interface DayHeadersProps {
+  testId?: string
+  isoDate: string
+}
+
+const useStyles = makeStyles({
+  dayHeader: {
+    position: 'sticky',
+    zIndex: 2,
+    borderLeft: '1px solid #C4C4C4',
+    borderBottom: 'none',
+    textAlign: 'center'
+  },
+  spaceHeader: {
+    border: 'none'
+  }
+})
+const DayHeaders = (props: DayHeadersProps) => {
+  const { start } = getPrepWeeklyDateRange(props.isoDate + 'T00:00:00-07:00')
```

Comment:
The 'T00:00:00-07:00' was a hack I added just to move us forward, I don't think this is a good solution. At the very least this should probably live in a function somewhere - but ideally we need to have a better way to set the date.